src/domain/communication/vc-interaction/vc.interaction.entity.ts

```diff
@@ -18,4 +22,7 @@ export class VcInteraction extends BaseAlkemioEntity implements IVcInteraction {
 
   @Column('char', { length: UUID_LENGTH, nullable: false })
   virtualContributorID!: string;
+
+  @Column('simple-json')
```

Comment:
If you put a default, it maybe a good idea to put it as decorator DB level and auto-generate it to the migration, will spare us trouble in the future.

Reply:
Mysql is not allowing a default value on DB level.